lib/praxis/types/multipart_array.rb

```diff
@@ -150,6 +148,67 @@ def self.example(context=Attributor::DEFAULT_ROOT_CONTEXT, **options)
         example
       end
 
+
+      def self.describe(shallow=true, example: nil)
+        type_name = Attributor.type_name(self)
```

Comment:
yes, but we could call that, to get the bits we need for our hash though...better than replicate the code here and have yet another place that might need to be changed if we change the "type" schema...